### ISSUE TITLE
Add the ability to program alarms

### ISSUE THREAD

**Issue by WindohsCrasher** [pascalopitz/unoffical-sonos-controller-for-linux#24]
It would be really great if you could add this feature :)

**Comment by pascalopitz** [comment]
Agreed ... This and basic music library management are coming next.

**Comment by WindohsCrasher** [comment]
Thank you :)

Am 24. Juli 2017 02:39:19 MESZ schrieb Pascal Opitz <notifications@github.com>:
>Agreed ... This and basic music library management are coming next.
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/pascalopitz/unoffical-sonos-controller-for-linux/issues/24#issuecomment-317294016

-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9 Mail gesendet.

**Comment by pascalopitz** [comment]
Ooops, wrong button ;)

**Comment by pascalopitz** [comment]
I made a bit of a start on this, but it's ... complicated.  It will require a whole browsing panel to pick the alarm media, i.e. a playlist.

One of my tasks is a complete refactor of the media browser panel anyways. I am looking to de-couple services, library and search from each other, but also have a bit of a project going with browsing and playing local files. I will try and separate the media browser bit out to it could be used to pick the media source as well.
